### ISSUE TITLE
configuration for systemd?

### ISSUE THREAD

**Issue by vigilancetech-com** [notqmail/notqmail#177]
Is there a recommended configuraton for running it purely under systemd (e.g. no daemontools)?

**Comment by mbhangui** [comment]
You could use something like this as a systemd unit file
```
[Unit]
Description=qmail mail delivery daemon
After=network.target

[Service]
ExecStart=/var/qmail/bin/qmail-start ./Maildir/ splogger qmail
Environment=PATH=/var/qmail/bin:${PATH}
Restart=always

[Install]
WantedBy=multi-user.target
```

**Comment by schmonz** [comment]
Merged in #114 and shipped in [1.08](https://github.com/notqmail/notqmail/wiki/notqmail-1.08).
